tests/console/consoletest_setup.pm

```diff
@@ -70,16 +70,14 @@ sub run() {
     script_run "ps axf > /tmp/psaxf.log";
     script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup.txt";
 
-    # Just after the setup: let's see the network configuration
```

Comment:
we also need it in each succeeded run, please do not remove it or only uploading when the test failed.